### DISCUSSION TITLE
Homepage Dashboard

### DISCUSSION THREAD

**Discussion by richardmoizeau** [openmsupply/openmsupply-client#1258] [Ideas]
This is just to record an idea that does not require immediate action. Dashboard follows the same UI as in Trello:

### Lists and cards

Definition:
- List = a list of cards/metrics. Currently we have: Inbound Shipment, Outbound Shipment & Stock
- Card (or metric): a scorecard with a metric related to the list it belongs to

In order not to have too many lists in the future, I would suggest to create list for each of the main menus (at least in the short term):

**Distribution:**

Number of Pending O/Shipments
Number of pending (customer) Requisitions

**Replenishment:**

number of Inbound Shipments (today / this week)
number of pending internal orders

**Inventory:**

number of expiring batch numbers
number of items with less than 3 months of stock
days since last stocktake
etc...

### Placeholder blue charts

I know it is controversial ;-) but I must insist of the fact that the "placeholder" chart (in blue) are misleading. If I have 0 inbound shipments today, the charts seems to indicate otherwise. I would suggest replacing them by relevant icons.

### Quick actions buttons

Should not only allow to create a new transaction (eg. a new Inbound Shipment) but also to simply view them. If a metric indicates that I have 4 pending I/S, the most likely user behaviour would be to access the list rather than creating a new one.

**Comment by adamdewey** [comment]
Hey @richardmoizeau - interesting, do you mean that the user is able to move cards around like Trello?

Yeah, agree that the blue charts are nice and pretty, especially when there isn't much on the dashboard currently, but a little misleading / unhelpful. 

Perhaps a smaller version of the graphic could be used as an icon next to the page title if we didn't want to drop it entirely?
